inbc-program/inbc/command/config_command.py

```diff
@@ -67,13 +67,14 @@ def __init__(self, broker: IBroker) -> None:
        """
         super().__init__(broker, 'get')
 
-    def trigger_manifest(self, args: Any, topic: str = CONFIG_CHANNEL + CONFIG_GET):
+    def trigger_manifest(self, args: Any, topic: str) -> None:
         """Trigger the command-line utility tool to invoke command.
 
         @param args: arguments from user
         @param topic: MQTT topic
         """
-        super().trigger_manifest(args, CONFIG_CHANNEL + CONFIG_GET)
+        channel = TC_INSTALL_CHANNEL if args.nohddl else CONFIG_CHANNEL + CONFIG_GET
+        super().trigger_manifest(args, channel)
```

Comment:
To reuse the code better and eliminate setting the channel in each of these, you can move the channel = TC_INSTALL_CHANNEL if args.nohddl else x into the super.triger_manifest and just pass the channel name when there isn't a args.nohddl tag included.

Reply:
Closing this as discussed